package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nvd3",
-  "version": "1.8.4-dev",
+  "version": "1.8.5-dev",
```

Comment:
I might be wrong, but as far as I remember, all previous new features increased the version, but I can change it back :)